UI/NativeApp.cpp

```diff
@@ -756,7 +786,11 @@ void NativeInit(int argc, const char *argv[], const char *savegame_dir, const ch
 
 	DEBUG_LOG(SYSTEM, "ScreenManager!");
 	g_screenManager = new ScreenManager();
+#if PPSSPP_PLATFORM(UWP) && !defined(__LIBRETRO__)
+	if (g_Config.memStickDirectory.empty() || (g_Config.bFirstRun && isLocalState(g_Config.memStickDirectory.ToString()))) {
```

Comment:
Since UWP controls this (`g_Config.memStickDirectory = Path(memstickFolderW);`), why not just set it to blank there if this case matches?

I guess the intention here is to always show this prompt on first run, right?

-[Unknown]

Reply:
Yes, I think if `g_Config.bFirstRun` is enough here, I don't remember why I made the other checks